api/src/shared/helpers.py

```diff
@@ -114,6 +116,51 @@ def parse_kv_str(kv_str):
         raise QueryArgumentError(repr(e), 422)
     return all_entities
 
+def log(*args):
+    """ intends to be used as a wrapper before an endpoint returns
+    to log information to the console and file using app.logger
+
+    calling with 2 arguments:
+        the first argument is the returning object from the endpoint
+        the second argument is the status code
+        
+    calling with 1 argument:
+        the single argument is a 2-element tuple:
```

Comment:
How often do you think we'll pass a tuple?

Reply:
I think that's the case where we return directly from `modify_entity`, which returns a tuple, and wrapped with the `log` function, we have a tuple being passed to the `log` function. Maybe we can destructure it, like `log(*modify_entity(...))`?

Reply:
Actually, `modify_entity` will return an already `jsonified` Response, which can't be passed to the `logged_response` because it will try to `jsonify` it again. Should I try to inspect whether the object passed in is already `jsonified` in `logged_response`, or should I simply implement the update without `modify_entity`?